shadowsocks-windows.bat

```diff
@@ -7,46 +7,14 @@ set TARGET_DIR=%CD%\x86_64-pc-windows-msvc
 echo Building libsodium
 msbuild libsodium\builds\msvc\vs2019\libsodium.sln /t:Clean /p:Configuration=StaticRelease /p:Platform=x64
 msbuild libsodium\builds\msvc\vs2019\libsodium.sln /t:Build /p:Configuration=StaticRelease /p:Platform=x64
-copy /y .\libsodium\bin\x64\Release\v142\static\libsodium.lib %BUILD_DIR%\sodium.lib
-
-echo Building openssl
-cd openssl
-  set OPENSSL_CONFIG=enable-capieng ^
-    enable-rfc3779 ^
-    no-ssl3 ^
-    no-camellia ^
-    no-cast ^
-    no-dso ^
-    no-idea ^
-    no-shared ^
-    no-seed
-
-  perl Configure VC-WIN64A ^
-    --prefix=%cd%\..\msvc-openssl ^
-    --openssldir=%cd%\..\msvc-openssl ^
-    %OPENSSL_CONFIG% ^
-    -FIWindows.h
-
-  nmake clean
-  nmake build_generated
-  nmake build_libs_nodep
-  copy /y libssl.lib %BUILD_DIR%
-  copy /y libcrypto.lib %BUILD_DIR%
-  mkdir %BUILD_DIR%\include\openssl
-  copy /y include\openssl\opensslconf.h %BUILD_DIR%\include\openssl\
-  copy /y include\openssl\opensslv.h %BUILD_DIR%\include\openssl\
-cd ..
+copy /y .\libsodium\bin\x64\Release\v142\static\libsodium.lib %BUILD_DIR%\
```

Comment:
I had to change the target filename here. When it was named only `sodium.lib` the Rust compiler complained it could not find the library. This fixed it.